dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml

```diff
@@ -0,0 +1,85 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - cohort_date
+          - client_id
```

Comment:
Do you also want to check here when the length of client_id is different than expected?